tools/uniprot_rest_interface/test-data/test2_retrieve.gff

```diff
@@ -1,16 +1,7 @@
 ##gff-version 3
 ##sequence-region S0DS17 1 369
-S0DS17	UniProtKB	Chain	1	369	.	.	.	ID=PRO_0000437163;Note=Cytochrome P450 momooxygenase apf8	
```

Comment:
Mh, another rebase? You can also merge master into this branch or simply close it and create a new PR. Thanks Dili!